tools/firtool/firtool.cpp

```diff
@@ -109,6 +109,11 @@ static cl::opt<bool>
                 cl::desc("run the width inference pass on firrtl"),
                 cl::init(true));
 
+static cl::opt<bool>
+    inferResets("infer-resets",
+                cl::desc("run the reset inference pass on firrtl"),
+                cl::init(false));
```

Comment:
I'm in favor of turning this on by default. We know of circuits that will fail to compile without this and I don't think going through an experimental period is necessary with something that is tightly following the SFC implementation. (Also, you've got a massive amount of testing here that makes me comfortable getting this into the pipeline.)

```suggestion
                cl::init(true));
```